pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -532,6 +536,10 @@ private void logBrokerResponse(long requestId, String query, RequestStatistics r
           requestStatistics.getReduceTimeMillis(), brokerResponse.getExceptionsSize(), serverStats.getServerStats(),
           brokerResponse.getOfflineThreadCpuTimeNs(), brokerResponse.getRealtimeThreadCpuTimeNs(),
           StringUtils.substring(query, 0, _queryLogLength));
+      if (totalTimeMs > _brokerSlowQueryLatencyThreshold) {
+        LOGGER.warn("Slow query: requestId={}, table={}, timeMs={}", requestId,
```

Comment:
Do we really need to put this extra line? We are basically logging the same thing twice when the total time ms is larger than the threshold. In practice, we can grep `timeMs > 500` to get the list of slow queries pretty easily.

Reply:
For manual debugging purposes, this won't add much value. But this can be useful for automatic tools as I commented below.